builder/docker/docker_test.go

```diff
@@ -0,0 +1 @@
+package docker
```

Comment:
MY TESTS ARE GONE! They must have been a victim of my bad rebase the other day. Hmmm... I'll fix that, or delete the file if there are no tests.